### ISSUE TITLE
Freecam

### ISSUE THREAD

**Issue by exm316** [Da0ne/DZMods#12]
Not sure if anything can be done, but whilst using freecam and searching for players they cant be seen unless freecam is initiated in the area of the player!

Examples:

Admin teleports to player and then uses freecam you can follow them easily but not see any loot until the player interacts with it.

Admin knows a player is at NWAF but initiates freecam whilst in Grishino and 'fly' to NWAF - player cannot be seen or found!

Just thought I would mention

**Comment by Da0ne** [comment]
>Not sure if anything can be done, but whilst using freecam and searching for players they cant be seen unless freecam is initiated in the area of the player! ...

@exm316 yes there is an issue with updating player network bubble. The camera can only render other players within 1k radius. You have to be in they general area to render. At the moment I can’t change that, will do when I start adding client side support to the mod.

**Comment by exm316** [comment]
Cheers kind sir, much appreciated
